bcda/auth/middleware_test.go

```diff
@@ -114,36 +113,6 @@ func (s *MiddlewareTestSuite) TestReturn401WhenExpiredToken() {
 	assert.Nil(s.T(), err)
 }
 
-// unit test
-func (s *MiddlewareTestSuite) TestRequireTokenAuthReturn401WhenInvalidToken() {
```

Comment:
This test is no longer needed because RequireTokenAuth does not do token validation anymore. The user scenario here (invalid token) is still covered by the following method:

- TestReturn400WhenInvalidTokenAuthWithInvalidSignature

Technically this 401 error was unreachable because the user should run into the 400 error from ParseToken before RequireTokenAuth is ever called.